zebra-chain/src/transaction/sighash.rs

```diff
@@ -134,6 +135,7 @@ impl<'a> SigHasher<'a> {
             Transaction::V1 { .. } | Transaction::V2 { .. } => unreachable!(ZIP143_EXPLANATION),
             Transaction::V3 { .. } => OVERWINTER_VERSION_GROUP_ID,
             Transaction::V4 { .. } => SAPLING_VERSION_GROUP_ID,
+            Transaction::V5 { .. } => ORCHARD_VERSION_GROUP_ID,
```

Comment:
Maybe the version group ID should become a method on `Transaction`, because this code is duplicated here and in transaction serialization.

Let's do this refactor in a follow-up?

Reply:
This follow-up is now #1828.